File: mlkit-pose-detection-accurate/src/main/java/com/king/mlkit/vision/pose/accurate/AccuratePoseCameraScanActivity.java
package com.king.mlkit.vision.pose.accurate;

import androidx.annotation.Nullable;

import com.google.mlkit.vision.pose.Pose;
import com.king.mlkit.vision.camera.BaseCameraScanActivity;
import com.king.mlkit.vision.camera.analyze.Analyzer;
import com.king.mlkit.vision.pose.accurate.analyze.AccuratePoseDetectionAnalyzer;

/**
 * @author <a href="mailto:jenly1314@gmail.com">Jenly</a>
 */
public abstract class AccuratePoseCameraScanActivity extends BaseCameraScanActivity<Pose> {

    @Nullable
    @Override
    public Analyzer<Pose> createAnalyzer() {
        return new AccuratePoseDetectionAnalyzer();
    }
}

File: mlkit-pose-detection-accurate/src/main/java/com/king/mlkit/vision/pose/accurate/AccuratePoseCameraScanActivity.kt
package com.king.mlkit.vision.pose.accurate

import com.google.mlkit.vision.pose.Pose
import com.king.camera.scan.BaseCameraScanActivity
import com.king.camera.scan.analyze.Analyzer
import com.king.mlkit.vision.pose.accurate.analyze.AccuratePoseDetectionAnalyzer

abstract class AccuratePoseCameraScanActivity : BaseCameraScanActivity<Pose>() {
    override fun createAnalyzer(): Analyzer<Pose>? = AccuratePoseDetectionAnalyzer()
}
